packages/runtime-common/virtual-network.ts

```diff
@@ -233,7 +233,8 @@ async function withRetries(
     } catch (err: any) {
       if (
         (globalThis as any).__environment !== 'test' ||
-        !baseRealm.inRealm(url) ||
+        (!baseRealm.inRealm(url) &&
+          !url.href.startsWith('https://boxel-icons.boxel.ai/')) ||
```

Comment:
i started seeing lots of spurious `fetch failed` CI errors for the icons lately...